modules/claims_api/config/schemas/v2/526.json

```diff
@@ -1044,6 +1044,11 @@
       "type": "boolean",
       "description": "Claimant certifies and authorizes release of information.",
       "default": false
+    },
+    "claimNotes": {
+      "type": "string",
+      "description": "An optional notes section.",
+      "maxLength": 4000
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Can this get put back in (same with modules/claims_api/config/schemas/v2/generate_pdf_526.json)